microcosm/loaders/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
+Configuration loading
+
+A configuration loader is any function that accepts `Metadata` and
+returns a `dict` (or `Configuration` model).
+
+Configuration might be loaded from a file, from environment variables,
+or from an external service.
+
+"""
+from microcosm.loaders.compose import (  # noqa: F401
+    load_each,
+)
+from microcosm.loaders.environment import (  # noqa: F401
+    load_from_environ,
+    load_from_environ_as_json,
+)
+from microcosm.loaders.keys import (  # noqa: F401
+    expand_config,
+)
+from microcosm.loaders.settings import (  # noqa: F401
+    load_from_json_file,
+)
+
+
+def load_from_dict(dct=None, **kwargs):
```

Comment:
Parties of me wants this in its own module, but I suspect no one will ever look at it so it's probably fine...

Reply:
I know what you mean. I couldn't decide where to put it, so here we are.